### ISSUE TITLE
Accessibility feedback on RSECon26 website

### ISSUE THREAD

**Issue by stefpiatek** [socrse/edia#69]
Belatedly tracking feedback for [RSECon26 website](https://rsecon26.society-rse.org/). Any feedback would be welcome, as well as links to external reports

**Comment by alexhambley** [comment]
(Copied from Slack comment)
Had a brief look and there's a few very minor things on the home page (not looked elsewhere):
- In the footer the company number link just reads, using a screen reader, as "1182455" which lacks context for screen reader users navigating by links (link should read something like "company number: 1182455",
- The "Deadline for submissions" banner sits outside a landmark (e.g nav landmark). Matters as it's orphaned if someone is navigating by landmark,
- Screen reader reads out Date and Location and the other eyebrow components as "slash slash date", "slash slash location", "slash slash vision". Creates some unnecessary noise and cognitive load.

**Comment by stefpiatek** [comment]
Done in https://github.com/socrse/rsecon26-website/issues/191
